geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/z2/package.scala

```diff
@@ -16,9 +16,9 @@ package object z2 {
 
   case class Z2IndexValues(sfc: Z2SFC,
                            geometries: FilterValues[Geometry],
-                           bounds: Seq[(Double, Double, Double, Double)])
+                           spatialBounds: Seq[(Double, Double, Double, Double)]) extends SpatialIndexValues
 
   case class XZ2IndexValues(sfc: XZ2SFC,
                             geometries: FilterValues[Geometry],
-                            bounds: Seq[(Double, Double, Double, Double)])
+                            spatialBounds: Seq[(Double, Double, Double, Double)]) extends SpatialIndexValues
```

Comment:
not sure if this is in the 'public api', but we could leave the `bounds` the same, mark it deprecated, and add in an `override def spatialBounds = bounds` method to fulfill the new SpatialIndexValues trait

Reply:
Good catch.  Just pushed a fix for this.